paddle/fluid/framework/tensor_util.cc

```diff
@@ -433,10 +489,9 @@ class AnyVisitor : public boost::static_visitor<bool> {
 
   bool GetResult(const framework::Tensor& out,
                  const platform::NPUPlace& npu) const {
-    PADDLE_THROW(platform::errors::Unimplemented(
-        "Not supported on place (%s) ",
-        npu));
-    //return GetResultHelper(out, npu);
+    PADDLE_THROW(
+        platform::errors::Unimplemented("Not supported on place (%s) ", npu));
+    // return GetResultHelper(out, npu);
```

Comment:
Should be refined!

Reply:
The modification is only for compilation issue, if this function is needed, it will be refined.